### DISCUSSION TITLE
Polaris has landed in the Apache GitHub org!

### DISCUSSION THREAD

**Discussion by snazy** [apache/polaris#172] [Show and tell]
We are thrilled to announce that the Polaris GitHub repository is now officially accessible as https://github.com/apache/polaris !

**Comment by stevenzwu** [comment]
would the contribution guideline be updated? right now, it still requires Snowflake ICLA and CCLA.
https://github.com/apache/polaris/blob/main/CONTRIBUTING.md

**Reply by snazy** [reply]
We're on it - and all the other things that need to be done

**Reply by jbonofre** [reply]
I already created a PR adapting the first resources. I'm waiting approval from @takidau 😄
